maven_install.json

```diff
@@ -651,6 +651,18 @@
                 "sha256": "66fdef91e9739348df7a096aa384a5685f4e875584cce89386a7a47251c4d8e9",
                 "url": "https://repo1.maven.org/maven2/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar"
             },
+            {
```

Comment:
Do we have to include this here, or is there a way it can be supplemented at runtime? We currently run tests in CI against both JRuby 9.2.x and 9.3.x.

Reply:
Good call, I'll modify rules_ruby to export the jruby jar